docs/index.html

```diff
@@ -20,14 +22,12 @@
   </style>
 </head>
 <body>
-
-
 <div id="app"></div>
 <script>
   window.$docsify = {
     homepage: 'chapters/01_introduction.md',
-    logo: 'img/booster-logo.png',
```

Comment:
In case we have removed the logo image, we should delete the png file too. Don't you think?

Reply:
The logo is used by the Readme.md file.

Reply:
Beware the fact that the logo png could be used in the Github Readme or contrib pages when you see them from Github repository